factory/installScripts/geckodriver/install-geckodriver-version.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/node
+const { spawn } = require('child_process')
+
+const geckodriverVersion = process.argv.slice(2)[0]
+
+if (!geckodriverVersion) {
+  console.log('No geckodriver version provided, skipping geckodriver install')
+  process.exit(0)
+}
+
+// See https://firefox-source-docs.mozilla.org/testing/geckodriver/Support.html for compatibility matrix
+// geckodriver < 0.34.0 only supports up to Firefox 120, which is no longer supported by Cypress
+const MIN_GECKO = '0.34.0'
+
+if (geckodriverVersion < MIN_GECKO) {
+  console.log(`geckodriver version ${geckodriverVersion} provided, minimum version ${MIN_GECKO} required, skipping geckodriver install`)
```

Comment:
Would `console.error` be more appropriate? I think we use `log` in similar error contexts so likely a no.

```suggestion
  console.error(`geckodriver version ${geckodriverVersion} provided, minimum version ${MIN_GECKO} required, skipping geckodriver install`)
```

Reply:
The risk with using `error` is that it kills the whole build. I've gone with `log` for consistency and to make sure that the build doesn't get killed if the version is too low. Simply skipping and logging the fact is less intrusive.

In general the build process does not do consistency checks across all parameters. It's left to the user to make sure they choose sensible combinations. See comments under https://github.com/cypress-io/cypress-docker-images/tree/master/factory#version-testing